src/main/java/uk/gov/justice/hmpps/prison/service/MovementsService.java

```diff
@@ -418,7 +418,7 @@ private static OffenderMovement transformToOffenderMovement(final ExternalMoveme
             .build();
     }
 
-    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER"})
+    @VerifyAgencyAccess(overrideRoles = {"SYSTEM_USER"})
```

Comment:
there is no bookingId passed in - so should us VerifyAgencyAccess (for the agencyId)